lego/apps/email/tasks.py

```diff
@@ -23,6 +23,10 @@
 log = get_logger()
 
 
+def add_source_to_url(url):
+    return f"{url}?source=WeeklyMail"
```

Comment:
You _could_ also add `utm_campaign=Email` (case sensitive) so that we can see how many use email in general and not just the weeklyMail.